examples/Sql/HiveDataFrame/Program.cs

```diff
@@ -24,7 +24,10 @@ static void Main(string[] args)
             var sparkConf = new SparkConf();
             var sparkContext = new SparkContext(sparkConf);
             var hiveContext = new HiveContext(sparkContext);
-            var peopleDataFrame = hiveContext.Read().Json(Path.Combine(Environment.CurrentDirectory, @"data\people.json"));
+            
+            // please give the people.json file path or just use data/people.json under this folder as input
+            var jsonFilePath = args.Length > 0 ? args[0] : Path.Combine(Environment.CurrentDirectory, @"data/people.json");
```

Comment:
To avoid confusion, I suggest removing the hard coded path and rely on args[0].
In instructions, you can point out the data set to use by adding link to https://github.com/Microsoft/Mobius/blob/master/examples/Sql/HiveDataFrame/data/people.json and also all supported file path formats in both Wndows and Linux